front/sssscs/src/certs/CertSummaryTable.tsx

```diff
@@ -1,12 +1,27 @@
-import { TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody } from "@mui/material";
-import { CertificateSummaryDTO, subjectToCellStr } from "./CertService";
+import { TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody, IconButton } from "@mui/material";
+import DownloadIcon from '@mui/icons-material/Download';
+import { CertService, CertificateSummaryDTO, subjectToCellStr } from "./CertService";
+import { AxiosError, AxiosResponse } from "axios";
 
 export interface CertSummaryTableProps {
     summaries: CertificateSummaryDTO[],
     onClickRow?: (row: number) => void,
 }
 
 export const CertSummaryTable = (props: CertSummaryTableProps) => {
+    const downloadCert = (id: number) => {
+        CertService.download(id)
+            .then((res: AxiosResponse<ArrayBuffer>) => {
+                const url = window.URL.createObjectURL(new Blob([res.data]));
+                const link = document.createElement("a");
+                link.href = url;
+                link.setAttribute("download", id + ".cert");
```

Comment:
`.crt` instead of `.cert`.

Reply:
https://github.com/magley/sssscs/pull/23/commits/52b32756d0def255dc8be07291f7d9bc7e06426d